src/app/components/GoalScorerDisplay.tsx

```diff
@@ -2,14 +2,11 @@ import React, { SyntheticEvent, Dispatch, useState } from "react";
 import { TeamScore } from "./TeamScorer";
 
 type Props = {
-  teamAScoreObject: TeamScore;
-  setTeamAScoreObject: Dispatch<React.SetStateAction<TeamScore>>;
+  teamScoreObject: TeamScore;
+  setTeamScoreObject: Dispatch<React.SetStateAction<TeamScore>>;
 };
 
-function GoalScorerDisplay({
-  teamAScoreObject,
-  setTeamAScoreObject,
-}: Props) {
+function GoalScorerDisplay({ teamScoreObject, setTeamScoreObject }: Props) {
   const inputRef = React.createRef<HTMLInputElement>();
```

Comment:
I didn't explain this properly before but we could also use the useState hook as above in `GameSetup.tsx`, and then use its onChange handler to update it and then refer to this in the `updateScorer` function. But since we also need to have a handle on the parent element (i.e. the form) it is easier to use a ref (which essentially gives you a handle to a DOM element which stays constant across renders). So with this we are able to easily access its value as well as its parent node to get the id. If we just went the `useState` route as suggested above, getting the id would need a bit more fiddling.